src/analyzeRxns.m

```diff
@@ -18,9 +18,11 @@
 
 %find all product producers
 makesProd = [];
-h = waitbar(0,['finding ',product,' producing strains']);
+%h = waitbar(0,['finding ',product,' producing strains']);
+h = showprogress(0,['finding ',product,' producing strains']);

```

Comment:
Please follow the coding style guide: `h = showprogress(0, ['finding ', product, ' producing strains']);`